keyboards/niu_mini/readme.md

```diff
@@ -11,6 +10,10 @@ Hardware Availability: [KBDFans](https://kbdfans.myshopify.com/products/niu-mini
 
 Make example for this keyboard (after setting up your build environment):
 
-    make niu_mini:default:dfu
+    make niu_mini:default
 
-See [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information.
+Flashing example for this keyboard (using [QMK Toolbox](https://docs.qmk.fm/#/newbs_flashing?id=flashing-your-keyboard-with-qmk-toolbox) or [via the command line](https://docs.qmk.fm/#/newbs_flashing?id=flash-your-keyboard-from-the-command-line)):
```

Comment:
The `:flash` target has nothing to do with the Toolbox, so I'm not sure why it's mentioned here? Perhaps this could be reworded to make it clearer they are different methods.

Reply:
Confused myself thinking about the template update :sweat_smile: